R/plot_var_compare.R

```diff
@@ -44,10 +47,9 @@ plot_var_compare = function(nc_file, field_file, var_name, fig_path = FALSE, res
   }
   
   #Pivot observed into table
-  
   x = as.numeric(as.POSIXct(data$DateTime))
   y = data$Depth
-  z = data$Observed_wTemp
+  z = data[,paste0('Observed_', var_name)]
```

Comment:
didn't know that `data[, paste0('Observed','_temp')]` is the same as `data[[paste0('Observed_', var_name)]]`.